src/state/slices/preferencesSlice/preferencesSlice.ts

```diff
@@ -115,6 +116,7 @@ const initialState: Preferences = {
     ThorchainLending: getConfig().REACT_APP_FEATURE_THORCHAIN_LENDING,
     LedgerWallet: getConfig().REACT_APP_FEATURE_LEDGER_WALLET,
     MultiHopTrades: getConfig().REACT_APP_FEATURE_MULTI_HOP_TRADES,
+    ThorswapLongtail: getConfig().REACT_APP_FEATURE_THORSWAP_LONGTAIL,
```

Comment:
as discussed in dm, lets rename `Thorswap*` to  `ThorchainSwap*`

Reply:
https://github.com/shapeshift/web/pull/5638/commits/27a1c79332b85efd53dde708777cf2687aec5322